examples/statsd.py

```diff
@@ -14,19 +15,8 @@ class SimpleHandler(statsd.StatsdMixin, web.RequestHandler):
 
     """
 
-    @gen.coroutine
-    def prepare(self):
-        maybe_future = super(SimpleHandler, self).prepare()
-        if concurrent.is_future(maybe_future):
-            yield maybe_future
-
-        if 'Correlation-ID' in self.request.headers:
-            self.set_metric_tag('correlation_id',
```

Comment:
Excellent!
With set_metric_tag() gone, prepare() is no longer necessary. Thanks for simplifying.